client/src/screens/availability/Requests.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import { View } from 'react-native';
+import SelectMultiple from 'react-native-select-multiple';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Icon from 'react-native-vector-icons/FontAwesome';
+
+import { setSelectedRequests } from '../../state/availability.actions';
+import requestsData from '../../fixtures/groups';
+
+class Requests extends Component {
+  static navigationOptions = () => ({
+    title: 'Availability',
```

Comment:
I suggest making the title 'Requests' - this seems to be pretty standard for iOS UI at least by looking through the Settings app.

Reply:
Changes the tabIcon when changing the title - I'll look to a work around.